draft-fossati-tls-attestation.md

```diff
@@ -1061,6 +1075,22 @@ the TLS server will terminate the exchange.
 
 TBD.
 
+# Privacy Considerations {#priv-cons}
+
+In background-check mode, the Verifier has access to detailed information about the Attester TLS peer's TCB through Evidence.
```

Comment:
I find the "TLS peer" suffix somewhat distracting, and I'm not sure if it's strictly needed since it should be fairly clear from the context that the attester and RP are the TLS peers. How strongly do you feel about keeping it in?